numpy/linalg/lapack_litemodule.c

```diff
@@ -17,7 +17,7 @@ typedef struct { float r, i; } f2c_complex;
 typedef struct { double r, i; } f2c_doublecomplex;
 /* typedef long int (*L_fp)(); */
 
-int FNAME(MAIN_)() { return 0; }
+void FNAME(MAIN_)() { return 0; }
```

Comment:
The return type should be `int`, what needs fixing is `FNAME(MAIN_)(void)` (or whatever is correct).